runtime/gc_include/ObjectAccessBarrierAPI.hpp

```diff
@@ -2668,6 +2668,11 @@ class MM_ObjectAccessBarrierAPI
 		}
 	}
 
+	VMINLINE j9flatobject_t inlineIndexableFlatObjectAddress(J9VMThread *vmThread, J9IndexableObject *arrayRef, J9Class *clazz, I_32 index)
+	{
+    	return (j9flatobject_t)vmThread->javaVM->memoryManagerFunctions->j9gc_objaccess_indexableFlatObjectAddress(vmThread, arrayRef, clazz, (I_32)index);
```

Comment:
@AidanHa  you need to add the function to the `memoryManagerFunctions` functions table in j9nonbuilder.

Reply:
You can basically do a search in the code base for `copyObjectFields` and copy what is done for that